bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx

```diff
@@ -71,6 +71,7 @@ class SettingsDropdown extends Component {
 
     this.state = {
       isSettingOpen: false,
+      isFullScreen: false,
```

Comment:
The state of `isFullScreen`, is already passed down as a prop from the container. I don't think we need to duplicate it here.